signac/contrib/filesystems.py

```diff
@@ -27,12 +28,19 @@
 GRIDFS_LARGE_FILE_WARNING_THRSHLD = int(1e9)  # 1GB
 FILESYSTEM_REGISTRY = dict()
 
+# this is here because of issues importing the same variable in
+# signac/__init__.py from the top level namespace
+__version__ = '1.2.0'
```

Comment:
Python 2.7 environments crash when I remove the current_version from the @deprecated decorators, so when we stop supporting python 2, this can be removed. Until then I think we should just leave it how it is.

Reply:
@tommy-waltmann So I did some research/testing. I think the optimal solution is to place the `__version__` variable in a separate `version.py` module within the package root. This makes it possible to import the variable both into the root and within the sub-packages.

I was not able to reproduce any issues with respect to Python 2 in combination with omitting this parameter, but either way, I would appreciate if you could apply above proposed change.